File: watermark-client/src/app/domains/PictureResponse.ts
export class PictureResponse {

    public url: string;
    public watermarkedUrl: string;
    public pictureId: number;

    constructor(url: string, pictureId: number, watermarkedUrl: string) {
        this.url = url;
        this.pictureId = pictureId;
        this.watermarkedUrl = watermarkedUrl;
      }
   
  }

File: watermark-api/src/main/java/com/watermark/watermarkapi/domains/PictureResponse.java
package com.watermark.watermarkapi.domains;

public class PictureResponse {

	private Integer pictureId;
	private String url;
	private String watermarkedUrl;

	public Integer getPictureId() {
		return pictureId;
	}

	public void setPictureId(Integer pictureId) {
		this.pictureId = pictureId;
	}

	public String getUrl() {
		return url;
	}

	public void setUrl(String url) {
		this.url = url;
	}

	public String getWatermarkedUrl() {
		return watermarkedUrl;
	}

	public void setWatermarkedUrl(String watermarkedUrl) {
		this.watermarkedUrl = watermarkedUrl;
	}

}
